### ISSUE TITLE
Enhancement request: Rule CORE-000289 reports false positive when --ORRES (char) is e.g. written as ".09", i.e. starting with a dot.

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#520]
I ran the latest version on the files of the Metadata Submission Guide 2.0.
In LB, there are some records for which LBORRES (which is "char") is e.g. written as ".09". For example record 1462 in lb.xpt.
The value in LBORNRLO (also "char") is written as ".32" and the value in LBORNRHI ("char") as "5". LBNRIND is populated with "LOW", so this all seems to be correct.
I do however get an error "LBORRES is not a continuous measurement or Empty, when LBORNRHI is not empty".
I presume this is due to ".09" not being recognized as a numeric value in a field (LBORRES) that is "char".
All other rows for which LBORRES value does not start with "." seems to be correctly evaluated.
![Unbenannt](https://github.com/cdisc-org/cdisc-rules-engine/assets/25853332/6dc03bc4-6e79-42ff-a199-b0c199b5f123)

**Comment by SFJohnson24** [comment]
[Rule(28).zip](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd1ltQ1E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f6ac2bb36e277709cc102a7fdcae3b8c0a4240ee/Rule%2828%29.zip) I ran this for .09 and .97 for LBORRES and received an issue raised for each.  The regex in this rule appears off if we want to allow leading decimals on numbers.  Right now it wont flag 0.09 but will flag .09.  ```^-?(0|[1-9]\d*)?(\.\d+)?$``` would fix this by allowing the leading decimal in front of numbers.

**Comment by JozefAerts** [comment]
Great! Thanks  @SFJohnson24 !
Regular expressions were never my strength ... (although I did work with Perl very long time ago).
I changed the regex in the YAML code (commented out the old one with some explanation), did all the tests again, and as these showed the expected results, re-published it.
Ticket can be closed.
Many thanks again!
